tests/jenesis/cmd/test_attach.py

```diff
@@ -0,0 +1,118 @@
+import os
+import shutil
+import subprocess
+import time
+from tempfile import mkdtemp
+from cosmpy.aerial.config import NetworkConfig
+from cosmpy.aerial.faucet import FaucetApi
+
+import pytest
+import toml
+from jenesis.config import Config
+
+
+#@pytest.mark.skip
+def test_attach():
+    """Test attach contract"""
+
+    # create first project and deploy a contract
+    network = "fetchai-testnet"
+    profile = "profile_1"
+
+    path = mkdtemp(prefix="jenesis-", suffix="-tmpl")
+
+    os.chdir(path)
+
+    # create project
+    Config.create_project("project_1", profile, network)
+    os.chdir(os.path.join(path, "project_1"))
+
+    template = "starter"
+    contract_name = "contract"
+
+    project_root = os.path.abspath(os.getcwd())
+
+    # add starter contract and update
+    contract = Config.add_contract(project_root, template, contract_name, None)
+    Config.update_project(os.getcwd(), profile, network, contract)
+
+    # compile contract and wait for it to finish
+    subprocess.run("jenesis compile", shell=True)
```

Comment:
Is it really necessary to compile in this test? It seems like we could just supply a hard-coded address of a contract on Dorado to attach to and then run a query and maybe execute and check the results.

Reply:
Yes, thats a better idea and will save time